lms/djangoapps/courseware/block_render.py

```diff
@@ -548,14 +548,10 @@ def inner_get_block(block):
         if staff_access:
             block_wrappers.append(partial(add_staff_markup, user, disable_staff_debug_info))
 
-    field_data = DateLookupFieldData(block._field_data, course_id, user)  # pylint: disable=protected-access
-    field_data = LmsFieldData(field_data, student_data)
```

Comment:
We already add these as wrappers in https://github.com/openedx/edx-platform/blob/3413277a63860a84e5fc5b6dea70a52efbd45296/lms/djangoapps/courseware/block_render.py#L658-L665.